samples/sample_vpp/src/sample_vpp.cpp

```diff
@@ -713,24 +713,24 @@ int main(int argc, msdk_char *argv[])
             if ( Params.use_extapi )
             {
                 mfxFrameSurface1 * out_surface = nullptr;
-                sts = frameProcessor.pmfxVPP->RunFrameVPPAsyncEx(
-                    pInSurf[nInStreamInd],
-                    pWorkSurf,
-                    //pExtData,
-                    &out_surface,
-                    &syncPoint);
-
-                while(MFX_WRN_DEVICE_BUSY == sts)
+                for (;;)
```

Comment:
can we change this place to `for(;;)` in a separate PR, please?

Reply:
https://github.com/Intel-Media-SDK/MediaSDK/pull/1696/files